extension/json/src/json_extension.cpp

```diff
@@ -0,0 +1,49 @@
+#include "json_extension.h"
+
+#include "catalog/catalog.h"
+#include "common/types/types.h"
+#include "json_functions.h"
+#include "json_scan.h"
+#include "main/client_context.h"
+#include "main/database.h"
+
+namespace kuzu {
+namespace json_extension {
+
+#define ADD_FUNC(FUNC_STRUCT)                                                                      \
```

Comment:
I would move this macro to extension_utils.h so every extension can utilize this function